### PR TITLE
Support latlonbnds

### PR DESCRIPTION
**Pull request by ernstdevreede**
Adds support for data where gid data cells are described by `lat_bnds` and `lon_bnds` variables.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+**Version 2.24.0 2024-06-27**
+- Adds support for grids where the data cells are defibe by lat_bnds and lon_bnds parameters describing the cell bounds.
```
Comment (maartenplieger) [inline NEWS.md]: are defined by by lat_bnds

File: adagucserverEC/CImageDataWriter.cpp [hunk 2]
```diff
@@ -723,6 +723,11 @@ int CImageDataWriter::getFeatureInfo(std::vector<CDataSource *> dataSources, int
         if (dataSources[d]->getDataObject(0)->cdfObject->getAttributeNE("ADAGUC_GEOJSON") != NULL) {
           openAll = true;
         }
+
+        if (dataSources[d]->getDataObject(0)->cdfObject->getAttributeNE("USE_ADAGUCCONVERTER") != NULL) {
```
Comment (maartenplieger) [inline adagucserverEC/CImageDataWriter.cpp]: Should `USE_ADAGUCCONVERTER` be renamed to `USE_ADAGUC_LATLONBNDS_CONVERTER` ?

File: tests/functional_test.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
   The adaguc-server executable is ran with various settings and configuration files.
   Results are checked against expected results and should not differ.
 """
+
 import unittest
 import sys
 from AdagucTests.TestWMSTimeSeries import TestWMSTimeSeries
```
Comment (maartenplieger) [inline tests/functional_test.py]: - Can you add a functional test demonstrating this dataset?
- Can you add documentation to https://github.com/KNMI/adaguc-server/tree/master/doc/format_standard ?

File: adagucserverEC/CImageDataWriter.cpp (hunk 2 above)
Comment (ernstdevreede) [reply adagucserverEC/CImageDataWriter.cpp]: Fixed

File: tests/functional_test.py (hunk 3 above)
Comment (ernstdevreede) [reply tests/functional_test.py]: Done

File: adagucserverEC/CConvertLatLonBndsData.cpp [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+    longitudeBnds->readData(CDF_DOUBLE, true);
+  }
+  if (latitudeBnds->data == nullptr) {
+    latitudeBnds->readData(CDF_DOUBLE, true);
+  }
+
+  // Detect minimum and maximum values
+  MinMax minMax;
+  minMax = getMinMax(((float *)irregularGridVar[0]->data), dataObjects[0]->hasNodataValue, (double)fltFill, irregularGridVar[0]->getSize());
+  CDBDebug("minMax %f %f", minMax.min, minMax.max);
```
Comment (maartenplieger) [inline adagucserverEC/CConvertLatLonBndsData.cpp:97]: Please ensure that there isn't too much logging going on.

Comment (ernstdevreede) [reply adagucserverEC/CConvertLatLonBndsData.cpp:97]: Done

**Review by maartenplieger — APPROVED** [review]

**Pull request merged**